src/ape_cache/__init__.py

```diff
@@ -0,0 +1,18 @@
+from ape import plugins
+from ape.api import PluginConfig
+
+from .query import CacheQueryProvider
+
+
+class CacheConfig(PluginConfig):
+    size: int = 1024**3  # 1gb
```

Comment:
should we add a validator which can recognize formats like "1gb" so a more readable value can be put into the yaml config?

Reply:
Perhaps. Also thinking if this makes more sense to live in to `ape cache prune --size 1gb` than in the config. If it's constantly pruning that will be really slow